convert2rhel/actions/package_updates.py

```diff
@@ -0,0 +1,81 @@
+# Copyright(C) 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import logging
+
+from convert2rhel import actions, pkgmanager
+from convert2rhel.pkghandler import get_total_packages_to_update
+from convert2rhel.repo import get_hardcoded_repofiles_dir
+from convert2rhel.systeminfo import system_info
+from convert2rhel.utils import ask_to_continue
+
+
+logger = logging.getLogger(__name__)
+
+
+class PackageUpdates(actions.Action):
+    id = "PACKAGE_UPDATES"
+
+    def run(self):
```

Comment:
Need to `super()` this or use the decorator.

Reply:
Thanks, just added it